gspread/worksheet.py

```diff
@@ -3201,3 +3202,51 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        source: str,
+        condition_type: ValidationConditionType,
+        *values: Iterable[Any],
+        **kwargs,
+    ) -> Any:
+        """Adds a data validation rule to any given range.
+
+        .. note::
+
+            ``condition_type`` values are explained here: `ConditionType`_
+
+            .._ConditionType: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/other#ConditionType
+
+
+        :param str source: The A1 notation of the source range to move
+        :param condition_type: The sort of condition to apply.
+        :param values: List of condition values or RelativeDate.
+        :type values: Any | RelativeDate
```

Comment:
Can we please add a couple or examples in the documentation like: `NUMBER_GREAtER`.

And/or (depending on the time to make it, a popular based condition like : ONE_OF_LIST

Reply:
Added in 811ff02b40a53757ca52d259b5b6ced59106365f